libgraphics/src/utils.cpp

```diff
@@ -1179,3 +1179,143 @@ bool MCGAffineTransformFromPoints(const MCGPoint p_src[3], const MCGPoint p_dst[
 }
 
 ////////////////////////////////////////////////////////////////////////////////
+
+bool MCGPaintCreateWithNone(MCGPaintRef& r_paint)
+{
+    r_paint = nullptr;
+    return true;
+}
+
+bool MCGPaintCreateWithSolidColor(MCGFloat p_red, MCGFloat p_green, MCGFloat p_blue, MCGFloat p_alpha, MCGPaintRef& r_paint)
+{
+    return MCGSolidColor::Create(p_red, p_green, p_blue, p_alpha, (MCGSolidColorRef&)r_paint);
+}
+
+bool MCGPaintCreateWithPattern(MCGImageRef p_image, MCGAffineTransform p_transform, MCGImageFilter p_filter, MCGPaintRef& r_paint)
+{
+    return MCGPattern::Create(p_image, p_transform, p_filter, (MCGPatternRef&)r_paint);
+}
+
+bool MCGPaintCreateWithGradient(MCGGradientFunction p_function, const MCGFloat* p_stops, const MCGColor* p_colors, uindex_t p_ramp_length, bool p_mirror, bool p_wrap, uint32_t p_repeats, MCGAffineTransform p_transform, MCGImageFilter p_filter, MCGPaintRef& r_paint)
```

Comment:
Should `MCGContextSetGradientPaint` from `context.cpp` be changed to use this function?